pythonforandroid/recipes/pygame/__init__.py

```diff
@@ -0,0 +1,56 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+from os.path import join
+from pythonforandroid.toolchain import current_directory
+
+
+class Pygame2Recipe(CompiledComponentsPythonRecipe):
+
+    version = '2.0.0-dev7'
+    url = 'https://github.com/pygame/pygame/archive/android-2.0.0-dev7.tar.gz'
```

Comment:
Usually the version should be configurable
```suggestion
    url = 'https://github.com/pygame/pygame/archive/android-{version}.tar.gz'
```